src/coreclr/jit/emitarm64.cpp

```diff
@@ -8985,7 +8985,8 @@ void emitter::emitIns_Call(EmitCallType          callType,
                            regNumber        xreg /* = REG_NA */,
                            unsigned         xmul /* = 0     */,
                            ssize_t          disp /* = 0     */,
-                           bool             isJump /* = false */)
+                           bool             isJump /* = false */,
+                           bool             isNoGCframe /* = false */)
```

Comment:
can you include the new parameter in the method description? for this file and in other files?

Reply:
Yes. A good point.

Reply:
I am also not sure if `isNoGCframe` is the best name. I could not come up with anything better at the time, but now I think `noSafePoint` might be better.

Reply:
I've renamed the parameter `noSafePoint`. I think it is better and easier to explain. I've also added comments about the parameter where we had any comments.